### PR TITLE
Convert preview command to Ruby.

### PR DESCRIPTION
**Pull request by noniq**
See http://lists.macromates.com/textmate/2016-July/039723.html for rationale. This rewrite should behave exactly like the previous version of the command.

I decided to use `TextMate::Markdown` because it seemed like a perfect fit, also it seems very easy to extend this code to support the `TM_MARKDOWN_FILTER` (as discussed on the mailing list).

### CODE REVIEW

**Comment by noniq** [comment]
Ping @infininight :)

File: Commands/Markdown preview.plist [hunk 1]
```diff
@@ -5,58 +5,73 @@
...
-      });
-    &lt;/script&gt;
-    &lt;script type=\"text/javascript\"    src=\"http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML\"&gt;
-    &lt;/script&gt;"
+  if n &gt; 0 &amp;&amp; m = lines[n].match(/&lt;(h\d|p|ul|li|blockquote|pre|div|img|code|table|tr)&gt;(.*)$/i)
+    lines[n] = "&lt;#{m[1]} id=\"scroll_to_here\" &gt;#{m[2]}"
+  end
+
+  io &lt;&lt; lines.join("\n")
+  if true || ENV["TM_MARKDOWN_MATHJAX"].to_i &gt; 0
```
Comment (sorbits) [inline Commands/Markdown preview.plist]: Is the `if true` perhaps remains from a test run?

Comment (noniq) [inline Commands/Markdown preview.plist]: Oh, of course, good catch! Will fix it immediately.

**Comment by sorbits** [comment]
Merged as 095ad165f8f4272a92cf17ed56ba24ca932a7d3d.

I squashed the commits into a single one and also did a minor rewording of your commit message, I hope you are OK with this change.

**Pull request closed without merging**